toolset/setup/linux/languages/jruby-1.7.sh

```diff
@@ -1,8 +1,11 @@
 #!/bin/bash
 
-fw_depends rvm
+fw_depends rvm java7
+
+# rvm stable [typically] only provides one version of jruby-1.7
+# update this when it changes
+JRUBY_VERSION="1.7.19"
```

Comment:
This should be moved down past the `RETCODE` check, and it needs to be included in the `.installed` file so that subsequent runs have access to he variable in the standard way.

Github is not allowing me to comment on the line I want, but I would expect it to look like this:

`echo "export JRUBY_VERSION=${JRUBY_VERSION}" > $IROOT/jruby-$JRUBY_VERSION.installed`

This change would also have to be made for the other Ruby language files.